libs/facilities/frontend/view/src/lib/components/issues/issues.page.ts

```diff
@@ -0,0 +1,20 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { RouterLink } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { XdBrowsePage } from '../browse';
+import { facilities } from '../facility';
+
+@Component({
+	selector: 'lib-issues',
+	standalone: true,
+	imports: [CommonModule, IxModule, RouterLink, XdBrowsePage],
+	templateUrl: './issues.page.html',
+	styleUrl: './issues.page.scss',
+	encapsulation: ViewEncapsulation.None,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class XdIssuesPage {
+	readonly issueFacilities = facilities.filter((facility) => facility.variant != 'success');
```

Comment:
nice readonly!

pls add
```suggestion
	protected readonly _issueFacilities = facilities.filter((facility) => facility.variant != 'success');
```

And again this should be a signal

Reply:
changed it